src/Grinder/appsettings.json

```diff
@@ -3,7 +3,7 @@
         "Token": "REPLACEME",
         "ChannelId": 0,
         "AdminUserId": 0,
-        "Socks5Proxy": {
+        "UNUSED_Socks5Proxy": {
```

Comment:
How's this now `UNUSED`? I can see you using it in code. It is optional, yes, but `UNUSED`?

Reply:
Deserialization hack to make whole field `Sock5Proxy` null (`AspNet.Configuration` can't deserialize to `Option<T>`)

It is essentially the same as deleting whole block, but I've decided to use `UNUSED_` prefix to remind it's here.

Reply:
https://github.com/Liminiens/grinder/pull/29/files#diff-604810767f0acd5f2918aef66e3ae8a2R48-R50

Reply:
So, it is actually used if it's there, right?

Reply:
Yep. Maybe prefix it with  `OPTIONAL_`? It's utterly a workaround to make whole section optional

Reply:
I can'see how prefixing it with any nonsensical prefix makes it "optional". In this state, it is, indeen, unused, but _why_ is it in this state? Why touch this config file at all?

Yeah, okay, the section is optional, I got it. But should we drop it from the example config?

Probably just mark it with `// optional` and that's it. AFAIK, .NET configuration system supports comments in JSON files.

Reply:
Oh right. Because it was hosted from Russia it definitely needed proxy to work :)

Azure West Europe DC network doesn't need proxy to connect to Telegram

We have to drop it from example config because it's not an example. It's hierarchically constructed where "example" appconfig.json is the first step and every other source is overwriting it (like environmental variables or Azure config).

And because this section exists in the very base config it won't be null ever.

Reply:
Alright, now I get it. Let's cover the whole `"Socks5Proxy"` section with a comment. Then the real users will be able to override it, and Azure mumbo jumbo won't have any problems, right?

Reply:
Well, if JSON comments will work it's definitely much better :) will try tomorrow

Reply:
@ForNeVeR done